### ISSUE TITLE
Incomplete chart drawing in logarithmic scale

### ISSUE THREAD

**Issue by andresghelarducci** [apache/superset#25829]
I created a stacked bar chart with logarithmic scale and sometimes some values are not displayed while present in tooltip

#### How to reproduce the bug

Create a stacked chart 

### Expected results
Stacked rectangle idrawn correctly

### Actual results

A stacked bar is missing one rectangle among the expected 3 ( see middle of the first image).

#### Screenshots
In the "ReviewFixed" stack I expect to see 3 rectangles but only the 2 are visible
![immagine](https://github.com/apache/superset/assets/44871257/8566b511-e1a2-4f58-b3e2-2a28dcce8dc4)

Same chart but in linear scale
![immagine](https://github.com/apache/superset/assets/44871257/a4c7befa-7733-4438-b54c-7e8b531bcc49)
Here  given the scale

### Environment

(please complete the following information):
Superset(downloaded 2 weeks ago) is run via docker compose 
- browser type and version: firefox
- superset version: image pulled is supersetxl:01
- python version: using version contained in the image 
- node.js version: using version contained in the image
- any feature flags active:

### Checklist

Make sure to follow these steps before submitting your issue - thank you!

- [ ] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [ ] I have reproduced the issue with at least the latest released version of superset.
- [ X] I have checked the issue tracker for the same issue and I haven't found one similar.

**Comment by rusackas** [comment]
Hi there. Can you provide some sample data to reproduce this issue (even if it's made up) as a CSV or a SQL query? Are you also able to say if the version of superset you downloaded was 3.0 or newer? Thanks :)

**Comment by justinpark** [comment]
FYI: This issue exists within ECharts (as reported [#19518](https://github.com/apache/echarts/pull/19518))

**Comment by rusackas** [comment]
@kasiazjc @kgabryje @michael-s-molina Curious your thoughts on the above proposal. You can basically have Stacking OR Logarithmic Axis, but not both. Curious for any UI ideas here, as this is an unusual thing in the design conditional controls, to disable one control and display a tooltiop/warning saying that it's off because another control is enabled somewhere else.

**Comment by rusackas** [comment]
The proposal in the thread — making logarithmic scale and stacking mutually exclusive options — seems like a reasonable path forward to avoid the rendering issue. If @kasiazjc, @kgabryje, or @michael-s-molina has input on this design decision, a comment would help unblock progress.

**Comment by rusackas** [comment]
Still waiting on UI input from @kasiazjc, @kgabryje, or @michael-s-molina on making stacking and log axis mutually exclusive. Anyone want to weigh in?

**Comment by rusackas** [comment]
Followed up on the ECharts side: their maintainers confirmed stacked bars can't be proportionally represented on a log axis and won't fix it upstream. Given that, the fix here is making `stack` and `logAxis` mutually exclusive in the control panel. Happy to see a PR for that.

**Comment by rusackas** [comment]
Opened an upstream fix for the disappearing-bar part of this: apache/echarts#21702. If that lands, the missing rectangles here should come back. The proportions caveat from the ECharts threads still stands, so we may still want a warning when stacking and log axis are combined... but hopefully we won't need to make them mutually exclusive after all.

**Comment by rusackas** [comment]
The echarts fix (apache/echarts#21702) is still open upstream, so this is still live. Given that, making `stack` and `logAxis` mutually exclusive in the control panel seems like the more reliable path rather than waiting on it. Anyone want to pick that up?
